contrib/automation_tests/test_cases/bottom_up_tab.py

```diff
@@ -5,43 +5,32 @@
 """
 import logging
 
+from core.common_controls import Table
 from core.orbit_e2e import E2ETestCase, find_control
 
 
 class VerifyHelloGgpBottomUpContents(E2ETestCase):
```

Comment:
Can you add a short class comment about what this expects and verifies?

Reply:
Done.